UI/window-basic-main-transitions.cpp

```diff
@@ -1770,8 +1770,7 @@ obs_data_array_t *OBSBasic::SaveTransitions()
 	return transitions;
 }
 
-void OBSBasic::LoadTransitions(obs_data_array_t *transitions,
-			       obs_load_source_cb cb, void *private_data)
```

Comment:
I feel like this change is out of scope for this PR, as we're changing a function signature, even though it's private. I'd be fine with this and its related changes as its own PR.